bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigUtil.java

```diff
@@ -56,25 +60,31 @@ public static Object normalizeType(Object value) {
      * BigDecimals, Strings and Booleans wherever a conversion of similar types was possible.
      *
      * However, it does not check for general correctness of types. This can be done using the
-     * ConfigDescriptionValidator.
+     * {@link ConfigDescriptionValidator}.
      *
-     * @param configuration
-     * @param configDescription
-     * @return
+     * @param configuration the configuration to be normalized (if null then the given configuration is returned)
```

Comment:
in fact this is a bit unclear because the given configuration is null in this case.... I will rephrase it